dcalc/DmpCeff.cc

```diff
@@ -92,13 +92,14 @@ gateModelRd(const LibertyCell *cell,
 	    double c1,
 	    const Pvt *pvt,
 	    bool pocv_enabled);
+template <typename Eval>
 static void
 newtonRaphson(const int max_iter,
 	      double x[],
 	      const int n,
 	      const double x_tol,
 	      // eval(state) is called to fill fvec and fjac.
-	      function<void ()> eval,
```

Comment:
Could the original slow-ness be because this is passed by value instead of `const &` ? I guess it depends on how much is captured with the call.

Maybe it helps to un-stall this PR if changing the signature to `const function<void()> &eval` could improve performance and don't require a callable template parameter.